---
title: 'Combining Geochemistry and Microbial Ecology to Understand Habitability and Processes in the Subsurface Terrestrial Biosphere'

event: AbSciCon 2019
event_url: https://agu.confex.com/agu/abscicon19/meetingapp.cgi/Paper/480519

location: 
address:
  street: 
  city: Seattle
  region: WA
  postcode: ''
  country: United States

summary: 
abstract: "**Introduction:** If there is or ever was life on an
extraterrestrial planetary body, evidence of that life
would likely be found below the surface, protected
from harsh exposed conditions. The deep terrestrial
subsurface is a key analog for subsurface settings on
other rocky planets. In Earth’s deep subsurface
biosphere (DSB), water, organic carbon and chemical
energy are often scarce. However, it has been shown
that archaea and bacteria are able to persist in the DSB
to at least 3.5 km below surface [1]. Understanding
how organisms persist, and what markers they may
leave behind, is critical in predicting which
extraterrestrial bodies may have harbored, or harbor,
similar life.
<br><br>
In the current study we investigated life down to
1.5 km below surface. Our portal to the DSB is a
legacy gold mine in Lead, South Dakota—now home
to the Deep Mine Microbial Observatory (DeMMO,
consisting of six sample sites, 800-4850 feet below
surface) hosted within a sulfide-rich, Paleoproterozoic
iron-formation [2]. Previous work by our group used
thermodynamic modeling from measured
geochemistry to predict energy-yielding metabolic
reactions and showed evidence for abundant exergonic
chemolithoautotrophic metabolisms [3]. Here, we build
upon that work by combining metagenomic sequences
with geochemical and 16S rRNA gene sequences,
culminating in 5 years of physical and biological data.
Synthesizing these datasets of different types and from
different time points provides a more holistic
understanding of how life persists in Earth’s DSB,
thereby enabling better prediction of what sorts of life
may be in the subsurface of extraterrestrial bodies.
<br><br>
Samples for this study were collected between
September, 2013 and September, 2018 from boreholes
located between 0-4850 feet (~1.5 km) below surface.
On each expedition, geochemical data were collected
concomitantly with DNA for 16S and/or metagenomic
sequencing.
<br><br>
**Results:** Preliminary 16S rRNA analyses revealed
a change in microbial community composition as a
function of depth, including a progression from
Proteobacteria-dominated and candidate phyladominated communities near the surface to Firmicutesdominated communities at depth. Nearly 20 candidate
phyla were identified in DeMMO fluids, which are
divisions for which we have no cultured representative.
Members of the OP3 (Ca. Omnitrophica) and
Hadesarcheaeota were particularly abundant in fluids
collected at 800 and 4850 feet deep, respectively.
Further investigation using whole genomic DNA
amplification- shotgun metagenomics- enabled complete
and near-complete genome reconstruction of surface
fluids flowing above DeMMO and fluids circulating
within the subsurface. We reconstructed nearly 500
genomes from 8 fluid samples. In some cases, the
metagenomic data recapitulated the 16S datasets,
enabling us to reconstruct full genomes for abundant
members, such as candidate phylum OP3. In other fluids
we were able to construct complete genomes for
members of the rare biosphere. For example, we
reconstructed the most complete genome for candidate
phylum NKB19 currently available [4]. Gene prediction
and metabolic mapping revealed that putative anaerobic
metabolisms, potentially using H2 as an electron donor,
were very common. The transformation of oxidized and
reduced sulfur and/or nitrogen species were also very
common potential energy sources among genomes.
Nearly half of the reconstructed genomes had all genes
necessary for carbon fixation via the relatively energy
inexpensive, and obligately anaerobic reductive acetylCoA pathway. In general, the phylogeny and
metabolisms of genomes collected from the surface
water above DeMMO were different from those found
in fluids collected 800-4850 feet below surface and
consisted mostly of aerobic and phototrophic
metabolisms. Conversely, the majority of microbes in
subsurface borehole fluids retain metabolic capability
for chemolithotrophic and autotrophic metabolisms.
These results, along with geochemical data and
hydrologic modeling, lead us to conclude that the deeply
circulating fluids at DeMMO have little if any continuity
with the overlying surface fluids, supporting the
possibility of subsurface life independent from the
surface on other planetary bodies.
<br><br>
**Broader implications:** Our study of microbial
communities in a terrestrial subsurface setting informs
the prediction about the location and possible
biosignatures of communities on extraterrestrial bodies.
Analyzing geochemical metadata with phylogenetic
diversity and metagenomic metabolic mapping may
enable us to predict what metabolic pathways would be
likely in the subsurface of other potentially habitable
planets such as Mars.
<br><br>
**References:** [1] Baker J. B. et al. (2003)
Environ. Microbiol., 5, 267–277. [2] Osburn M.
R. et al. Front. in Earth Sci (in review) [3] Osburn
M. R. et al. (2014) Front. in Microbiol., 5. [4]
Momper L. et al. Front. in Microbiol., (2018)"

# Talk start and end times.
#   End time can optionally be hidden by prefixing the line with `#`.
date: "2019-06-26T15:15:00Z"
date_end: "2019-06-26T15:30:00Z"
all_day: false

# Schedule page publish date (NOT talk date).
publishDate: "2019-06-01T00:00:00Z"

authors: 
- lily
- caitlin
- brittany
- ted
- maggie
tags: []

# Is this a featured talk? (true/false)
featured: false

# image:
#   caption: 'Image credit: [**Unsplash**](https://unsplash.com/photos/bzdhc5b3Bxs)'
#   focal_point: Right

links:
- icon: twitter
  icon_pack: fab
  name: Follow
  url: https://twitter.com/clorosexi
url_code: ""
url_pdf: ""
url_slides: ""
url_video: ""

# Markdown Slides (optional).
#   Associate this talk with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides = "example-slides"` references `content/slides/example-slides.md`.
#   Otherwise, set `slides = ""`.
slides: 

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects : [metagenomics]
---

